drivers/imu.py

```diff
@@ -0,0 +1,94 @@
+"""
+  Simple IMU data analysis
+"""
+
+
+import sys
+import os
+import inspect
+
+OSGAR_ROOT = os.path.realpath(os.path.abspath(os.path.join(os.path.split(inspect.getfile( inspect.currentframe() ))[0],"..")))
+if OSGAR_ROOT not in sys.path:
+    sys.path.insert(0, OSGAR_ROOT) # access to logger without installation
+
+
+import serial
+from threading import Thread, Event
+
+from lib.logger import LogWriter, LogReader
+from drivers.gps import checksum
+
+
+def parse_line(line):
+    assert line.startswith(b'$VNYMR'), line
+    assert b'*' in line, line
+    s = line.split(b'*')[0].split(b',')
+    assert len(s) == 13, s
+    arr = [float(x) for x in s[1:]]
+    return arr[:3], arr[3:6], arr[6:9], arr[9:]
```

Comment:
Is this NMEA parsing? I think it would make a great module in lib 😉

Reply:
Well, yes and no. The "parsing" is actually only the first line, and for that I do not consider it worth it (yet). I added bug fix for dGPS, where nice NMEA sentences are mixed with binary blobs and soon (with usage of "differential part") we will switch to parse binary blob.

I am also not sure if it is worth it to move `checksum()` to new file `../lib/nmea.py` if NMEA parsing itself is necessary _only_in drivers?? (and for that we would need import up and down as with `lib.logger`)